package.json

```diff
@@ -38,7 +38,9 @@
     "i18n:create-locales-list": "node src/locales/scripts/create-wp-locale-list"
   },
   "dependencies": {
+    "@babel/eslint-parser": "^7.16.5",
     "@nuxt/components": "^2.1.6",
+    "@nuxt/postcss8": "^1.1.3",
```

Comment:
Shouldn't `@nuxt/postcss8` be placed in dev dependencies? As was the tailwind module and is `autoprefixer`.